plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/flex/FlexDeployPreferencesTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google Inc.
```

Comment:
What happened is I deleted this file and created an entirely new file that happens to have the same file name. These files don't have anything common, and I didn't reuse any of the previous code. So I think 2017 is OK?